IM_calculation/scripts/calculate_ims.py

```diff
@@ -220,9 +220,17 @@ def main():
     station_names = args.station_names
     if args.advanced_ims is not None:
         components = advanced_IM_factory.COMP_DICT.keys()
+        extra_flags = []
+        # TODO: this could utilize check_rotd(), but adds extra complexity handling enums..
+        # TODO: more elegant solution could be desired if more IMs need special care
+        if "Burks_Baker_2013" in args.advanced_ims and set(
+            ["rotd50", "rotd100", "rotd100_50"]
+        ).intersection(args.components):
+            extra_flags.append("--rotd")
         advanced_im_config = advanced_IM_factory.advanced_im_config(
-            args.advanced_ims, args.advanced_im_config, args.OpenSees_path
+            args.advanced_ims, args.advanced_im_config, args.OpenSees_path, extra_flags
         )
+
```

Comment:
Adding --rotd to extra flag means that it will be added to the call of every model, not just Burks_Baker. So if two models are given and rotd_50 is specified, this will crash.
I think that if you want rotd_50 you should only be running with burks_baker, or you run without rotd_50, and throw an exception if they try to do both

Reply:
oh, that's a good point.

Reply:
@jpa198  Hopefully you like my new solution better.

Reply:
I think that rotd_50 flag should be set as part of the implementation of the adv_im or if you actually do need to turn it on / off advanced im_config seems like the more appropriate place to store it.

Reply:
Or configuration in the Model folder itself. Advanced IMs are made to be quite self contained, rather than having configuration passed through IM_calc. The reason for that is that each model is quite unique and has different configuration based on the IM.

Reply:
We discussed it and decided against it 0 advanced im_config is not an ideal place to turn things on and off. At the moment, SDI is the only advanced IM that needs rotd calculated. We could certainly devise a more elegant solution later with more IMs in need of rotd coming along. Currently we have a use case where this issue was blocking and switching to run.py was not an ideal alternative (lack of multi-core support, orchestration of the workflow, support for binary waveform etc).